membrane_curvature/base.py

```diff
@@ -54,15 +56,42 @@ class MembraneCurvature(AnalysisBase):
     results.gaussian_curvature : ndarray
         Gaussian curvature associated to the surface.
         Arrays of shape (`n_frames`, `n_x_bins`, `n_y_bins`)
+    results.average_z_surface : ndarray 
+        Average of the array elements in `z_surface`. 
+        Each array has shape (`n_x_bins`, `n_y_bins`)
+    results.average_mean_curvature : ndarray 
+        Average of the array elements in `mean_curvature`.
+        Each array has shape (`n_x_bins`, `n_y_bins`)
+    results.average_gaussian_curvature: ndarray 
+        Average of the array elements in `gaussian_curvature`.
+        Each array has shape (`n_x_bins`, `n_y_bins`)
+
+
+    Notes
+    -----
+    The derived surface and calculated curvatures are available in the
+    ``.results``::atttributes. 
+
+    The attribute :attr:`MembraneCurvature.z_surface`
```

Comment:
Isn't this inside `results`? So
```suggestion
    The attribute :attr:`MembraneCurvature.results.z_surface`
```

If you only want to display `z_surface` use the tilde
```reST
    The attribute :attr:`~MembraneCurvature.results.z_surface`
```